composer.json

```diff
@@ -56,7 +57,7 @@
         "static-analysis": [
             "vendor/bin/phpstan analyse"
         ],
-        "tests": [
+        "unit-tests": [
```

Comment:
Hm.. Generally this is not appreciated by the maintainers.

Is there any reason you decided to change this?

Reply:
👍 it would be best to avoid changing lines that are not related to the pull request

Reply:
My bad, left it by accident. Anyways, reference `@unit-tests` does not correlate to `"tests"` so it's not started with rest of scripts.
Reverting change.